app/cdap/components/ColumnViewPanel/components/ColumnViewWidget/index.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography, Input, IconButton } from '@material-ui/core';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import React, { Fragment, useRef, useState } from 'react';
+import DrawerWidgetHeading from 'components/ColumnViewPanel/components/ColumnViewWidget/DrawerWidgetHeading';
+import SearchIcon from '@material-ui/icons/SearchOutlined';
+import styled from 'styled-components';
+import { ReactNode } from 'react';
+
+export interface IColumnViewWidget {
+  headingText: ReactNode;
+  closeClickHandler: () => void;
+  searchedTermHandler: (searchedTerm: string) => void;
```

Comment:
Handlers should be named `onSomething`. E.g. `onSearchTermChanged`

Reply:
renamed handlers